madmom/utils/__init__.py

```diff
@@ -419,6 +426,97 @@ def quantize_events(events, fps, length=None, shift=None):
     return quantized
 
 
+def quantize_notes(notes, fps, length=None, num_notes=None, shift=None,
+                   offset=None, velocity=None):
+    """
+    Quantize the notes with the given resolution.
```

Comment:
I think the description needs to be elaborated. I would expect that "quantizing the notes" would "snap" their start and end points to a grid. This function, however, does something else. I'm not sure how to describe this concisely...

Reply:
Right, I didn't see it that way. Maybe `expand_notes` would be a more appropriate name?

Reply:
Added a more descriptive docstring.